de_docs/getting-started/user-guides/agenda-voting.md

```diff
@@ -0,0 +1,140 @@
+
+For example, issuing `dcrctl --wallet setvotechoice sdiffalgorithm yes` results in the following changes to `dcrctl --wallet getvotechoices`:
+
+```
+{
+  "version": 4,
+  "choices": [
+    {
+      "agendaid": "sdiffalgorithm",
+      "agendadescription": "Change stake difficulty algorithm as defined in DCP0001",
+      "choiceid": "yes",
+      "choicedescription": "change to the new algorithm"
+    },
+    {
+      "agendaid": "lnsupport",
+      "agendadescription": "Request developers begin work on Lightning Network (LN) integration",
+      "choiceid": "abstain",
+      "choicedescription": "abstain from voting"
+    }
+  ]
+}
+```
+
+---
+
+## **Stakepool-voting**
+
+If your Stakepool has updated to the latest stakepool software, you will find a "Voting" page in the navigation menu with dropdown options for each agenda. After you've chosen how you want your tickets to vote, simply press the "Update Voting Preferences" to save your votechoices. Below you'll find an image of the votechoices for vote version 4.
+
+<img src="/img/stakepool-voting-page.png">
+
+---
+
+## **Verifying Your Vote**
+
+The easiest method to track your how your tickets actually voted is to use the [block explorer](https://mainnet.decred.org) to view each of your votes.
+
+The block explorer has been updated to display YES and NO votes for each agenda. If a ticket voted abstain, it will not display anything. The first ticket in this example image voted "ABSTAIN" for both agendas, the second ticket voted "YES" for both agendas, and the third ticket voted "YES" for the lnsupport agenda and "ABSTAIN" for the sdiffalgorithm agenda.
+
+<img src="/img/verify_block-explorer-votes.png">
+
+You can also manually verify your vote by inputting the txid for the ticket purchase transaction into the block explorer or clicking the link for a Voted Ticket on your stakepool's website. Below is a quick walkthrough, starting from a stakepool's website. The red numbers of each image corresponds to the step number.
+
+1. Go to your stakepool's Tickets page, open the Voted Tickets section and click the txid for the ticket you want to inspect. This will open the block explorer to the ticket purchase transaction. (Figure 3)
+2. Press the '+' button to view the transaction details. (Figure 4)
+3. Press the '>' on the 1st output of the transaction (the 'stakesubsmission' output) to open the block explorer to the actual voting transaction. (Figure 4)
+4. The vote transaction should already be expanded, if not, press the '+' button. (Figure 5)
+5. Locate the second output of the transaction. The vote for this ticket is shown within the first 4 digits of the OP_RETURN. The 6th digit shows the vote version. In the images below, a version 4 vote voted "01" *in hexadecimal* (aka 0x01). (Figure 5)
```

Comment:
If you want to be completely accurate, the version is actually all 8 digits starting from the 6th.  It is a little-endian encoded unsigned 32-bit integer.